lua/wikis/callofduty/MainPageLayout/data.lua

```diff
@@ -0,0 +1,233 @@
+local WidgetUtil = Lua.import('Module:Widget/Util')
+
+local CONTENT = {
+	usefulArticles = {
+		heading = 'Useful Articles',
+		body = '{{Liquipedia:Useful Articles}}',
+		padding = true,
+		boxid = 1503,
+	},
+	wantToHelp = {
+		heading = 'Want To Help?',
+		body = '{{Liquipedia:Want_to_help}}',
+		padding = true,
+		boxid = 1504,
+	},
+	transfers = {
+		heading = 'Transfers',
+		body = TransfersList{rumours = true, limit = 10},
+		boxid = 1509,
+	},
+	thisDay = {
+		heading = ThisDayWidgets.Title(),
+		body = ThisDayWidgets.Content{
+			birthdayListPage = 'Birthday list'
+		},
+		padding = true,
+		boxid = 1510,
+	},
+	specialEvents = {
+		noPanel = true,
+		body = '{{Liquipedia:Special Event}}',
+	},
+	liquipediaApp = {
+		heading = 'Download the Liquipedia App',
+		padding = true,
+		body = '{{Liquipedia:App}}',
+		boxid = 1505,
+	},
+	filterButtons = {
+		noPanel = true,
+		body = Div{
+			css = { width = '100%', ['margin-bottom'] = '8px' },
+			children = { FilterButtonsWidget() }
+		}
+	},
+	matches = {
+		heading = 'Matches',
+		body = WidgetUtil.collect(
+			MatchTickerContainer{},
+			Div{
+				css = {
+					['white-space'] = 'nowrap',
+					display = 'block',
+					margin = '0 10px',
+					['font-size'] = '15px',
+					['font-style'] = 'italic',
+					['text-align'] = 'center',
+				},
+				children = { Link{ children = 'See more matches', link = 'Liquipedia:Matches'} }
+			}
+		),
+		padding = true,
+		boxid = 1507,
+		panelAttributes = {
+			['data-switch-group-container'] = 'countdown',
+		},
+	},
+	tournaments = {
+		heading = 'Tournaments',
+		body = TournamentsTicker{
+			upcomingDays = 30,
+			completedDays = 20,
+			displayGameIcons = true
+		},
+		padding = true,
+		boxid = 1508,
+	},
+}
+
+return {
+	banner = {
+		lightmode = 'COD logo lightmode.svg',
+		darkmode = 'COD logo darkmode.svg',
+	},
+	metadesc = 'The Call of Duty (COD) esports wiki covering everything from players, teams and transfers, ' ..
+		'to tournaments and results, maps and games.',
+	title = 'Call of Duty',
+	navigation = {
+		{
+			file = 'Elevate at CODM World Championship 2024.jpg',
+			title = 'Teams',
+			link = 'Portal:Teams',
+			count = {
+				method = 'LPDB',
+				table = 'team',
+			},
+		},
+		{
+			file = 'Seattle Surge at EWC 2024.jpg',
+			title = 'Players',
+			link = 'Portal:Players',
+			count = {
+				method = 'LPDB',
+				table = 'player',
+			},
+		},
+		{
+			file = 'Warzone Trophy at EWC 2024.jpg',
+			title = 'Tournaments',
+			link = 'Portal:Tournaments',
+			count = {
+				method = 'LPDB',
+				table = 'tournament',
+			},
+		},
+		{
+			file = 'Insight at EWC 2024.jpg',
+			title = 'Transfers',
+			link = 'Portal:Transfers',
+			count = {
+				method = 'LPDB',
+				table = 'transfer',
+			},
+		},
+		{
+			file = 'COD Infinite Warfare Wallpaper.jpg',
+			title = 'Games',
+			link = 'Portal:Games',
+			count = {
+				method = 'CATEGORY',
+				category = 'Games',
+			},
```

Comment:
Why is this not using LPDB query?

Reply:
I"m not sure but for unknown reason  the number **6** does appear, which actually matches that theres only 6 page with infobox games at the moment. Maybe Should I change the category method?
![image](https://github.com/user-attachments/assets/8508b0b5-959c-4f13-a9ba-d35a8f59bfb5)

Reply:
Well, you're gonna get 6 regardless